server/context/conf.js

```diff
@@ -40,6 +40,11 @@ if (IS_COVERAGE_ACTIVE) {
     configuration = JSON.parse(configurationString);
     Log.info('[Configuration] ', configuration);
   } catch (e) {
+    if (e instanceof SyntaxError) {
+      let errMsg = `Error: ${coverageFile} is not a valid JSON`;
+      console.error(errMsg, e);
+      Log.error(errMsg, e, e.stack);
```

Comment:
Sorry, didn't see your comment. No problem. I'll make that change and a rebase as soon as finish opening a new issue I've found :wink: